components/list/list.js

```diff
@@ -56,6 +56,9 @@ class List extends PageableMixin(SelectionMixin(LitElement)) {
 			:host {
 				display: block;
 			}
+			:host > div {
```

Comment:
Targeting the `div` works right now, but the list component delegates rendering of the paging container to the pageable mixin. If someone modifies that rendering method to include a wrapper `div` for some reason without knowing that consumers have selectors targeting them, they would get `2px` of whitespace when they shouldn't. It would be safer to add a class to the `div` and target that (or the `role` attribute) to avoid the risk of that happening.